NSSAServer.java

```diff
@@ -74,6 +75,148 @@ public NSSAServer(){
 
 
     /**
+<<<<<<< HEAD
```

Comment:
When in cases you get this: <<< Head, that means there was a merge conflict (I may have accidentally left it in). But in cases of this, you have segments of <<< and  ==== >>>> which separates the changes. The part in the Head section are the recent changes and below the ==== is usually the previous code. In this case where there's nothing below it, you would need to remove the <<< Head and ==== symbols from the code.